ext/pdo/tests/bug_36798.phpt

```diff
@@ -26,18 +26,24 @@ require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
 $db = PDOTest::factory();
 
 @$db->exec("SET NAMES 'LATIN1'"); // needed for PostgreSQL
-$db->exec("CREATE TABLE test (id INTEGER)");
-$db->exec("INSERT INTO test (id) VALUES (1)");
+$db->exec("CREATE TABLE test36798 (id INTEGER)");
+$db->exec("INSERT INTO test36798 (id) VALUES (1)");
 
-$stmt = $db->prepare("SELECT '�' as test FROM test WHERE id = :id");
+$stmt = $db->prepare("SELECT 'Ã' as test FROM test36798 WHERE id = :id");
 $stmt->execute(array(":id" => 1));
 
 $row = $stmt->fetch(PDO::FETCH_NUM);
 var_dump( $row );
 
+?>
+--CLEAN--
+<?php
+require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
+$db = PDOTest::factory();
+PDOTest::dropTableIfExists($db, "test36798");
 ?>
 --EXPECT--
 array(1) {
   [0]=>
-  string(1) "�"
+  string(1) "�"
```

Comment:
You may need to edit this test not in an editor but something like ``nano`` as it looks like the encoding thing is important, or write the string as a byte sequence and compare the row result.